vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,220 @@
-# Business Objective
-
+# Overview and Objective
+ 
 The Entando Process Driven Applications plugin is built to provide a
-rich and full featured user experience to allow for the management and
-completion of business processes and automation. The UX is delivered a
-general purpose UX layer built using micro frontends that can be
-utilized for any business process or task engine. Customers drive the UX
-layer by utilizing a pre-developed Entando integration or by
-implementing a set of interfaces on the server side. The backend is a
-Spring Boot microservice that provides a pluggable interface allowing
-for the injection of any underlying process or automation toolkit. The
-interfaces and steps for creating a new PDA backend implementation are
-provided below. This document provides an overview of the Entando
-Process Driven Apps (PDA) plugin. The initial section covers the
-business functionality and user facing micro frontends. The second
-section covers the architecture and the points of extension for the PDA
-plugin. The third provides a detailed technical overview for developers
-to add new functionality to a PDA implementation.
-
-# Terminology
-
+rich and full-featured user experience, facilitating the management and
+completion of business processes and automation.
```

Comment:
I know this is all still in progress ... 'facilitating the management and completion of business processes and automation.'  not sure what this means. Maybe some real-life features would be good? It might make complete sense to developers, but would it be helpful to have something for lay people?

Reply:
Buzzword compliance, mostly, but I think it makes sense if someone is familiar with process automation. Not necessarily even a developer in that case.